app/components/dashboard/show/collections_list_component.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= render Elements::HeaderComponent.new('h3', classes: ['mb-4'], value: label) %>
+
+<% collections.each do |collection| %>
+  <%= render Elements::HeaderComponent.new('h4', value: link_to(collection.title, "/collections/#{collection.druid}")) %>
```

Comment:
Please use a path helper instead of hardcoding. (If there isn't a path yet then don't link.)

Reply:
Actually, I realized in this case we don't have a collection show page yet, so I'm making this just the text of the title. I'll write up a ticket for the collection show page and updating this link